config/test.exs

```diff
@@ -138,7 +138,7 @@ config :omg_watcher,
   block_getter_loops_interval_ms: 50,
   # NOTE `exit_processor_sla_margin` can't be made shorter. At 3 it sometimes
   # causes :unchallenged_exit because `geth --dev` is too fast
-  exit_processor_sla_margin: 5,
+  exit_processor_sla_seconds: 75,
```

Comment:
But this is test - prev we had 1 secs blocks - is it OK?

Reply:
yes yes 1 sec blocks. Updating!